metron-interface/metron-rest/README.md

```diff
@@ -353,6 +355,20 @@ Request and Response objects are JSON formatted.  The JSON schemas are available
       * searchRequest - Search request
   * Returns:
     * 200 - Search results
+    
```

Comment:
This isn't intended to be the source of truth for types that we support.  I would think any data type someone may need should be supported in some way.  This is restricted to (mostly) simple types and is driven by the need to format simple values in different ways.  Anything else besides simple types is categorized as "other".  If at any point there is a need to include more types this can easily be extended.  

The Elasticsearch API gives you either a number or string (only string in the case of the REST API) so we need more information about the type in Elasticsearch to format it correctly.  Solr is similar.  I will take another pass at the documentation to make this more obvious.